File: DataSummitSolution/DataSummitUI/src/app/efModels/userInfoType.ts
import { UserInfo } from "./userInfo";

export class UserInfoType {

	UserInfoTypeId: number;
	Type: string;
	CreatedDate?: Date | string;
	UserId?: number;
	UserInfo: UserInfo[];

	constructor(userInfoTypeId?: number, type?: string, createdDate?: Date | string,
		userId?: number, userInfo?: UserInfo[])
	{
		this.UserInfoTypeId = userInfoTypeId || 0;
		this.Type = type || "";
		this.CreatedDate = createdDate || new Date(Date.now()) || null;
		this.UserId = userId || 0 || null;
		this.UserInfo = userInfo;
	}
}

File: DataSummitSolution/DataSummitModels/DB/UserInfoType.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace DataSummitModels.DB
{
    public partial class UserInfoType
    {
        public UserInfoType()
        {
            UserInfos = new HashSet<UserInfo>();
        }

        public byte UserInfoTypeId { get; set; }
        public string Type { get; set; }
        public DateTime? CreatedDate { get; set; }
        public long? UserId { get; set; }

        public virtual ICollection<UserInfo> UserInfos { get; set; }
    }
}
